src/main/resources/views/CommercialVehicles/Abandoned.hbs

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html>
+  <head>
+    <title id="documentType">{{documentType}}</title>
+    <style type="text/css" media="print, screen">
+        {{~> assets/stylesheets/version-1 this}}
+        {{~> assets/stylesheets/cvs-version-1 this}}
+        {{~> assets/stylesheets/Abandoned this}}
+    </style>
+  </head>
+  <body>
+    <!-- certificate -->
+    <div class="front-page">
+      <div class="certificate">
+        <div class="certificate__wrapper">
+
+          <!-- header -->
+          <div class="header">
+            <img src="{{root}}/assets/images/dvsa_crest.png" alt="DVSA" class="header__crest-img"/>
+            <div class="title-text">
+              <div class="bold-text" id="firstPageTitle">
+                Notification of Failure to Comply with the Conditions of Acceptance of a
+                  {{titleTextIncludingRollingHeaders}} ({{documentType}})
```

Comment:
Not sure if it's a massive issue, one for PO to confirm really but 
`{{titleTextIncludingRollingHeaders}} ({{documentType}})` is on a new line in the ticket, it appears to concat on line 2 in generated certs.

Reply:
@m-coslett what is your opinion ? get it exactly like the example ?

Reply:
the pdfs are from UX, so i'd guess thats what they're after. up to product though

Reply:
new code added to force it onto a new line. (done by putting the 3rd line into its own html div element